play-services-nearby-core-ui/src/main/res/xml/filepaths.xml

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
The filename of this file is a global resource name across microg. Thus, the name should be picked such that it is tied to the exposure notification or nearby module that it's used with instead of something very generic like "filepaths".